packages/haul-core/src/config/types.ts

```diff
@@ -36,7 +36,8 @@ export type BundleConfig = {
     Exclude<keyof MinifyOptions, 'sourceMap'>
   >;
   sourceMap?: boolean | 'inline';
-  dllDependencies?: string[];
+  dll?: boolean;
+  dependsOn?: string[];
```

Comment:
don't forget to update the docs

Reply:
the docs needs to be redone from scratch, so I'll do it at the end of this project